src/Features/CSharp/Portable/InlineDeclaration/CSharpInlineDeclarationCodeFixProvider.cs

```diff
@@ -84,33 +92,66 @@ public override Task RegisterCodeFixesAsync(CodeFixContext context)
 
             if (singleDeclarator)
             {
-                // This was a local statement with a single variable in it.  Just Remove 
+                // This was a local statement with a single variable in it.  Just Remove
                 // the entire local declaration statement.  Note that comments belonging to
                 // this local statement will be moved to be above the statement containing
-                // the out-var. 
+                // the out-var.
                 var localDeclarationStatement = (LocalDeclarationStatementSyntax)declaration.Parent;
                 var block = (BlockSyntax)localDeclarationStatement.Parent;
                 var declarationIndex = block.Statements.IndexOf(localDeclarationStatement);
 
-                if (declarationIndex > 0 &&
-                    sourceText.AreOnSameLine(block.Statements[declarationIndex - 1].GetLastToken(), localDeclarationStatement.GetFirstToken()))
+                bool TryFindPriorSurvivingStatementOnSameLine(out StatementSyntax statement)
```

Comment:
note:  in Roslyn 'Try' already implies 'can return null'.  So this could just be ```StatementSyntax TryFindPrior...()```.  So no need for out params and whatnot.